src/main/java/microsoft/exchange/webservices/data/core/ExchangeServiceBase.java

```diff
@@ -484,12 +484,6 @@ protected void setCustomUserAgent(String userAgent) {
    * @throws microsoft.exchange.webservices.data.exception.ServiceLocalException the service local exception
    */
   public void validate() throws ServiceLocalException {
```

Comment:
I think the fact that the method is empty doesn't mean that we should necessarily make it an abstract class.  I'll leave it as "default validation is no-op".